recipe/meta.yaml

```diff
@@ -82,12 +82,14 @@ outputs:
         - empy >=3.3,<3.4
         # static graphing
         - graphviz
+        - pillow
         # report-timings
         - pandas >=1.0,<2
         - matplotlib-base
+        # main loop plugins
+        - pympler
```

Comment:
This dependency was purposefully omitted before (something only we are likely to be interested in).

Reply:
Ah right

Reply:
Sadly there isn't a simple way of mapping between pypi optional dependencies and conda-forge outputs.

So far we've got a stripped-down bare-bones package (`cylc-flow-base`) and a bells-and-whistles package (`cylc-flow`). We can always add more as per-demand but I think these two should do for now.